models/Link.js

```diff
@@ -42,6 +42,13 @@ const LinkSchema = new mongoose.Schema(
       type: mongoose.Schema.Types.ObjectId,
       ref: "Profile",
     },
+    animation: {
+      type: String,
```

Comment:
I think an `enum` would be better using the config options and save the property name and not the value

Reply:
Here is an example https://github.com/EddieHubCommunity/LinkFree/blob/fc191d363f6b96c1a45f5c011f5e0204d7f7906c/models/Profile.js#L25-L28

Reply:
Do you need some help with this?

Reply:
yes please

Reply:
It looks good now but instead of the config object values, it should be the name

```diff
- "No Animation"
+ "none"
```